DQMServices/Components/plugins/MEtoEDMConverter.cc

```diff
@@ -81,6 +81,8 @@ MEtoEDMConverter::MEtoEDMConverter(const edm::ParameterSet & iPSet) :
   produces<MEtoEDM<TString>, edm::Transition::EndLuminosityBlock>(sName);
 
   consumesMany<DQMToken>();
+  consumesMany<DQMRunToken>();
```

Comment:
These need to be 
```
consumesMany<DQMToken, edm::InLumi>();
consumesMany<DQMRunToken, edm::InRun>();
```

Reply:
in fact, I guess we could just use `DQMToken` for both Run and Lumi MEs, and let the framework differentiate based on the `InLumi` and `InRun` flags.

Reply:
ok, let me try. _Maybe_ this fixes our problmes...

Reply:
obviously that would be too easy. Fixed, but #22281 continues...